src/contact.js

```diff
@@ -0,0 +1,25 @@
+import map from './images/map.png';
+
+const contactPage = () => {
+  const main = document.createElement('div');
+  main.setAttribute('class', 'main');
+
+  main.innerHTML = `
+    <div class="intro">
+    <h1>Book the Table</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua:</p>
+      <br>
+    <ul>
+      <li><strong>Contact Name: </strong> Bruce Wayne</li>
+      <li><strong>Phone number: </strong> +376456456</li>
+      <li><strong>Address: </strong> Street 123</li>
+      <li><strong>email: </strong> info@table.com</li>
+    </ul>
+  </div>
+  <img class="image" src= ${map} alt="">
```

Comment:
Let's make use of JS to generate the HTML structure.

This project requires us to use `document.createElement` and `document.appendChild` to appends HTML elements, just like you did in `line 4`.

we don't want the HTML structure to be hardcoded.

Reply:
Thanks for the suggestions offered. I follow them and remove the HTML injected code, and use exclusively DOM traversing.